demo/ota-Agent-Orchestrator/ota_demo.c

```diff
@@ -181,8 +177,11 @@ static void initMqttDownloader( AfrOtaJobDocumentFields_t *jobFields )
     numOfBlocksRemaining += ( jobFields->fileSize %
                             mqttFileDownloader_CONFIG_BLOCK_SIZE > 0 ) ? 1 : 0;
     currentFileId = jobFields->fileId;
-    currentBlockOffset = 0;
     totalBytesReceived = 0;
+    int bitmapSize = numOfBlocksRemaining/8;
+    bitmapSize += ( numOfBlocksRemaining % 8 > 0 ) ? 1 : 0;
```

Comment:
Not a blocker: I think we can coalesce this into one statement which is pretty standard.
```suggestion
    int bitmapSize = ( numOfBlocksRemaining + ( 8 - 1 ) ) / 8;
```

Reply:
I'm going to go with @AniruddhaKanhere's suggestion here. I _really don't like_ those macros in the old OTA agent. It takes something short and simple like the value 8 and aliases it with a macro for readability rather than fixing the core problem, unreadable code.